### ISSUE TITLE
is it possible to match an image from an external source?

### ISSUE THREAD

**Issue by avnerbarr** [paucarre/tiefvision#67]
I've trained my dataset withe my set of images. I was wondering if it is possible to find a match using an external image from a picture I've taken with a phone camera

**Comment by ICC-qi** [comment]
Yes, you can use **search.lua** in /src/torch/**11-similarity-searcher-cnn-file** like this
`$luajit search.lua test.jpg /home/.../A -dataFolder /home/.../B`
A is the folder where test.jpg is located in; B is the dataset of images to be compared, -dataFolder option is added by myself.

**Comment by ICC-qi** [comment]
If you don't add the new option -dataFolder. The default folder of images dataset is /tiefvision/src/torch/data/encoded-images. Don't forget to move your images there.
